app/src/main/java/ar/com/wolox/android/example/ui/auth/login/AuthPresenter.kt

```diff
@@ -10,10 +15,16 @@ import javax.inject.Inject
 
 class AuthPresenter @Inject constructor(
     private val loginRepository: LoginRepository,
+    private val context: Context,
```

Comment:
It's not recommended that the Presenter have access to Android Context. The main reason of doing this, is because when you're testing, leaving the presenter without a concept of Fragment, Activity, or Android Context, doesn't make you to create a complex mocking of system components. Also, implementing the context inside the presenter may produce unexpected behavior since the main users of this are Fragments, Activities, etc.